src/mon/PGMap.h

```diff
@@ -71,6 +71,10 @@ class PGMapDigest {
 
   mempool::pgmap::map<int64_t,interval_set<snapid_t>> purged_snaps;
 
+  bool use_legacy_per_pool_stats() const {
```

Comment:
IMO the name (and hence the subsequent usage) is confusing - it'ss either legacy or per-pool, not both. Currently the function returns true when per-pool stats to be enabled, but callers seem to treat that as legacy...

Reply:
agree, changing this to ``per_pool`` throughout